homework5/task2.py

```diff
@@ -0,0 +1,27 @@
+"""Создайте декоратор, который хранит результаты вызовы функции
+(за все время вызовов, не только текущий запуск программы)
+"""
+
+
+def dec(func_):
```

Comment:
here is your file:
[('lala',)][('lala',), ('lalalala',)][('lala',), ('lalalala',), ('lalalalalala',)]

but I see lala was called only once in code. Here I see 3 of them